data/channels.csv

```diff
@@ -13177,7 +13177,7 @@ KTVEDT1.us,KTVE-DT1,,NBC,,US,US-LA,Monroe,s/US-LA,eng,,FALSE,,,,https://www.myar
 KTVEDT2.us,KTVE-DT2,,FOX,,US,US-LA,Monroe,s/US-LA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f0/Fox_Networks_Group_US_logo.svg/512px-Fox_Networks_Group_US_logo.svg.png
 KTVEDT3.us,KTVE-DT3,,Laff,,US,US-AR,El Dorado,s/US-AR,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/1/1e/LAFF_logo.png
 KTVEDT4.us,KTVE-DT4,,Ion Mystery,,US,US-AR,El Dorado,s/US-AR,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/fb/ION_Mystery_logo.svg/512px-ION_Mystery_logo.svg.png
-KTVEthraa.kw,KTV Ethraa,,,,KW,,,c/KW,ara,,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/7uzAvQU.png
+KTVEthraa.kw,KTV Ethraa,,,,KW,,,c/KW,ara,,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/uOZOdqL.png

```

Comment:
Logo is too large (https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels).

```suggestion
KTVEthraa.kw,KTV Ethraa,,,,KW,,,c/KW,ara,,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/hqEbc49.png
```

Reply:
can't commit suggestion as I get the error: "This diff has recently been updated."